pkg/domain/podman/site.go

```diff
@@ -846,6 +863,8 @@ func (s *SiteHandler) prepareControllerDeployment(site *Site) *SkupperDeployment
 		endpoint = "/tmp/podman.sock"
 		volumeMounts[sockFile] = endpoint
 	}
+	memoryLimit, _ := strconv.ParseInt(site.ControllerOpts.MemoryLimit, 10, 64)
+	cpus, _ := strconv.Atoi(site.ControllerOpts.CpuLimit)
```

Comment:
@fgiorgetti could you help me understand why you are confident to not check these errors?

I do see above you may be setting them yourself e.g. `site.RouterOpts.MemoryLimit = strconv.FormatInt(c.MemoryLimit, 10)`. If that's always the case I'm not sure I understand why we'd want to involve string marshaling/unmarshaling at all.

Reply:
@c-kruse Actually I am missing an early validation of these values, which are validated somewhere else
on kubernetes sites. I will certainly add an early stage validation.

With regards to the other comment, that other validation is not needed, as it is just retrieving what is
currently applied to the container.

Thanks for finding it. I am going to update the PR once the validation is done.